sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryTableWithV2Filter.scala

```diff
@@ -30,7 +30,17 @@ class InMemoryTableWithV2Filter(
     schema: StructType,
     partitioning: Array[Transform],
     properties: util.Map[String, String])
-  extends InMemoryTable(name, schema, partitioning, properties) {
+  extends InMemoryBaseTable(name, schema, partitioning, properties) with SupportsDeleteV2 {
+
+  override def canDeleteWhere(filters: Array[Predicate]): Boolean = {
+    InMemoryTableWithV2Filter.supportsFilters(filters)
+  }
+
+  override def deleteWhere(filters: Array[Predicate]): Unit = dataMap.synchronized {
+    import org.apache.spark.sql.connector.catalog.CatalogV2Implicits.MultipartIdentifierHelper
+    dataMap --= InMemoryTableWithV2Filter
+      .filtersToKeys(dataMap.keys, partCols.map(_.toSeq.quoted), filters)
```

Comment:
`import ...InMemoryTableWithV2Filter._`
Then we can simplify the code.

Reply:
Importing the class vs. explicitly specifying the class doesn't seem to make much difference to me. I prefer to keep this as is.